spyder/locale/de/LC_MESSAGES/spyder.po

```diff
@@ -1205,8 +1205,8 @@ msgid "Appearance"
 msgstr "Erscheinungsbild"
 
 #: spyder/plugins/configdialog.py:1120 spyder/plugins/ipythonconsole.py:633
-msgid "Advanced Settings"
-msgstr "Erweiterte Einstellungen"
```

Comment:
My German is not what it could be, but I am quite sure that it should be "Einstellungen" as all nouns are capitalized in German.

Reply:
Thanks! We are reverting those changes

Reply:
Indeed :-) (my german is too bad nowadays)